src/NewTwitchApi/Resources/UsersApi.php

```diff
@@ -101,4 +101,23 @@ public function getActiveUserExtensions(string $bearer, string $userId = null):
 
         return $this->callApi('users/extensions', $bearer, $queryParamsMap);
     }
+
+    /**
+     * @throws GuzzleException
+     * @link https://dev.twitch.tv/docs/api/reference#create-user-follows
+     */
+    public function createUserFollow(string $bearer, string $fromId, string $toId, bool $notifications = null): ResponseInterface
+    {
+        $queryParamsMap = [];
+
+        $queryParamsMap[] = ['key' => 'from_id', 'value' => $fromId];
+
+        $queryParamsMap[] = ['key' => 'to_id', 'value' => $toId];
+
+        if ($notifications) {
```

Comment:
If I explicitly send `false`, then `allow_notifications` will not be in my query params. However, I would expect `allow_notifications=false`. In that case, I might as well have not passed in a parameter at all, because `false` is the same as `null`.

Should be this an `=== null` check instead?

You have a spec function for the true case. Please create the necessary spec functions to also describe and support any non-true cases (false, null, etc).